src/styles/fonts.scss

```diff
@@ -0,0 +1,69 @@
+$font-folder: "../../fonts";
+
+@function generate-src($family, $family-suffix) {
+    $src: null;
+    // $use-woff2, $use-woff, $use-ttf are dynamically set from environment variables via webpack.common.js, defaults to true
```

Comment:
I think the idea was that people could use the `.scss` files directly in their build if they wanted to. For that use case, I think it makes sense to [set the defaults](https://sass-lang.com/documentation/variables/#default-values) here:
```scss
$use-ttf: true !default;
$use-woff: true !default;
$use-woff2: true !default;
```
The user can then override these if they want, but there's a reasonable behavior if they don't.